bots/notify/src/main/java/org/openjdk/skara/bots/notify/RepositoryWorkItem.java

```diff
@@ -364,13 +368,26 @@ public Collection<WorkItem> run(Path scratchPath) {
             for (var ref : knownRefs) {
                 if (!hasBranchHistory) {
                     log.warning("No previous history found for any branch - resetting mark for '" + ref.name());
-                    for (var listener : listeners) {
-                        log.info("Resetting mark for branch '" + ref.name() + "' for listener '" + listener.name() + "'");
-                        history.setBranchHash(new Branch(ref.name()), listener.name(), ref.hash());
+                    int existingCommits = localRepo.commitCount();
+                    if (existingCommits <= NEW_REPOSITORY_COMMIT_THRESHOLD) {
```

Comment:
Unless the variable name is clarifying something, which it isn't here, then better just inline the call.